### PR TITLE
ci: enable aggregated release notes

### PR DESCRIPTION
**Pull request by VictorAvelar**
#### Summary

Use dotansimha/changesets-action as it includes the aggregate feature.

## Description

- Added step to get the version number to publish the release notes
- Changed the changeset/action to use a fork with the aggregate feature
- Enabled the aggregate release notes feature.
- Included script to get the latest version from one of the packages (took it from app-kit)

### CODE REVIEW

**Comment by CarlosCortizasCT** [comment]
Is there a way we can test this before merging so we check if it will work in this repo and how the release note would look like?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -57,13 +57,24 @@ jobs:
         env:
           CI: true
 
+      - name: Storing release version for changeset
+        id: release_version
+        run: echo "VALUE=$(./scripts/print_release_version.sh)" >> $GITHUB_OUTPUT
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
```
Comment (CarlosCortizasCT) [inline .github/workflows/release.yml:65]: I was wondering what do we need this env for? 🤔

File: scripts/print_release_version.sh
Comment (emmenko) [inline scripts/print_release_version.sh:1]: This file needs to be executable.

```
chmod 755 scripts/print_release_version.sh
```

File: .github/workflows/release.yml (hunk 1 above)
Comment (emmenko) [reply .github/workflows/release.yml:65]: It's needed for changeset (we execute `changeset version` in the script) as far as I know.

**Review by emmenko — COMMENTED** [review]
Thanks! 🙌

**Comment by emmenko** [comment]
> Is there a way we can test this before merging so we check if it will work in this repo and how the release note would look like?

Not really, we need to trigger a normal release to really find out.

**Review by emmenko — APPROVED** [review]
🥳

**Review by CarlosCortizasCT — APPROVED** [review]
👍

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -57,13 +57,24 @@ jobs:
...
+
       - name: Create Release Pull Request or Publish to npm
         id: changesets
-        uses: changesets/action@v1.4.4
+        # uses: changesets/action@v1.4.4
```
Comment (kark) [inline .github/workflows/release.yml:69]: nit: would that still be needed?

**Review by kark — APPROVED** [review]
Thanks for the first contribution here and for researching the changesets action fork 🎉 🙌

File: .github/workflows/release.yml (hunk 2 above)
Comment (VictorAvelar) [reply .github/workflows/release.yml:69]: If it works, then no. 😄

**Pull request merged**